test/verifier/verifier_test.go

```diff
@@ -248,7 +249,16 @@ func TestVerifier(t *testing.T) {
 
 					// Print verifier stats appearing on the last line of the log, e.g.
 					// 'processed 12248 insns (limit 1000000) ...'.
-					for n, p := range coll.Programs {
+					// Sort by program names for stable output.
+					names := make([]string, len(coll.Programs))
+					i := 0
+					for n := range coll.Programs {
+						names[i] = n
```

Comment:
Nit: Why don't we just do
```suggestion
						names = append(names, n)
```
That way we don't need the counter variable `i`

Reply:
If you use `names := make(string[], 0, len(coll.Programs))` you preallocate the correct capacity but the length of the slice will be 0 and `append` will start at index 0.